functable.c

```diff
@@ -358,10 +340,7 @@ static int init_functable(void) {
     FUNCTABLE_VERIFY_ASSIGN(ft, chunkmemset_safe);
     FUNCTABLE_VERIFY_ASSIGN(ft, compare256);
     FUNCTABLE_VERIFY_ASSIGN(ft, crc32);
-    FUNCTABLE_VERIFY_ASSIGN(ft, crc32_fold);
-    FUNCTABLE_VERIFY_ASSIGN(ft, crc32_fold_copy);
-    FUNCTABLE_VERIFY_ASSIGN(ft, crc32_fold_final);
-    FUNCTABLE_VERIFY_ASSIGN(ft, crc32_fold_reset);
+    FUNCTABLE_VERIFY_ASSIGN(ft, crc32_copy);
```

Comment:
`ft.crc32_copy` may be NULL.
Using `FUNCTABLE_VERIFY_ASSIGN` will produce the error:
```
Zlib-ng functable failed initialization!
```

https://github.com/zlib-ng/zlib-ng/actions/runs/20088522280/job/57631057600?pr=2048#step:18:43